drivers/memc/memc_bflb_bl61x.c

```diff
@@ -0,0 +1,399 @@
+/*
+ * Copyright (c) 2025 MASSDRIVER EI (massdriver.space)
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT bflb_bl61x_psram
+
+#include <zephyr/device.h>
+#include <zephyr/drivers/syscon.h>
+#include <zephyr/drivers/clock_control.h>
+#include <zephyr/dt-bindings/clock/bflb_bl61x_clock.h>
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(memc_bflb_bl61x, CONFIG_MEMC_LOG_LEVEL);
+
+#include <bouffalolab/bl61x/bflb_soc.h>
+#include <bouffalolab/bl61x/glb_reg.h>
+#include <bouffalolab/bl61x/psram_reg.h>
+
+#define EFUSE_DEV_INFOS_OFFSET 0x18
+#define EFUSE_PSRAM_SIZE_POS 24
+#define EFUSE_PSRAM_SIZE_MSK 3
+#define EFUSE_FLASH_SIZE_POS 26
+#define EFUSE_FLASH_SIZE_MSK 7
```

Comment:
It is not used in this driver (BFLB SDK grouped flash and RAM config), so may be ok to drop
```suggestion
```